core/src/main/java/com/linecorp/armeria/client/WebClient.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 LINE Corporation
+ * Copyright 2021 LINE Corporation
```

Comment:
nit: revert?
We don't change the year when we modify the code. 😉

Reply:
Done.